scout/server/blueprints/login/static/dick_tracy.svg

```diff
@@ -0,0 +1 @@
8 60" x="0px" y="0px"><title>Legal services_expand</title><path d="M46.3,37.523a6,6,0,0,0-3.263-4.757L36.085,29.22a.75.75,0,1,0-.682,1.336L42.352,34.1a4.543,4.543,0,0,1,2.46,3.627l1.7,9.651a.749.749,0,0,0,.737.62.734.734,0,0,0,.131-.012.748.748,0,0,0,.608-.868Z"/><path d="M12.674,29.547a.75.75,0,0,0-1.009-.327L4.717,32.766A5.981,5.981,0,0,0,1.46,37.487L.009,47.139a.747.747,0,0,0,.63.852A.714.714,0,0,0,.751,48a.749.749,0,0,0,.74-.639l1.456-9.687A4.512,4.512,0,0,1,5.4,34.1l6.949-3.546A.75.75,0,0,0,12.674,29.547Z"/><path d="M35.13,31.7l-2.8.177,5.285-7.52a.75.75,0,0,0-.057-.935,15.163,15.163,0,0,0-5.513-3.367A17,17,0,0,0,33.5,15c4.318-.788,7.5-2.1,7.5-3.729,0-.837-.579-2.057-5.588-2.985a.75.75,0,0,0-.273,1.475c3.318.615,4.277,1.325,4.364,1.485-.176.569-2.6,1.662-6.794,2.38a.735.735,0,0,0-.134.022,52.351,52.351,0,0,1-8.626.674c-9.488,0-15.279-2.183-15.557-3.014.089-.222,1.047-.932,4.366-1.547a.75.75,0,0,0-.274-1.475C7.481,9.218,6.9,10.438,6.9,11.275c0,1.633,3.184,2.941,7.5,3.729a17.031,17.031,0,0,0,1.436,5.005,15.234,15.234,0,0,0-5.646,3.416.75.75,0,0,0-.057.935l5.285,7.52-2.8-.177a.737.737,0,0,0-.724.425.748.748,0,0,0,.124.83l9.028,9.852v4.44a.75.75,0,0,0,1.5,0V42.518a.6.6,0,0,0-.014-.074c0-.021.005-.041,0-.063L20.39,30.814l2.994,3.509a.748.748,0,0,0,.57.264h0a.746.746,0,0,0,.57-.265l2.863-3.369L25.057,42.367c0,.024,0,.046,0,.069a.785.785,0,0,0-.015.082V47.25a.75.75,0,0,0,1.5,0V42.813l9.186-9.85a.749.749,0,0,0,.128-.832A.714.714,0,0,0,35.13,31.7ZM23.954,15.826a56.59,56.59,0,0,0,8.009-.569c-.755,4.834-3.971,9.664-8.009,9.664a6.182,6.182,0,0,1-4.276-1.947A.7.7,0,0,0,19.5,22.8a14.258,14.258,0,0,1-3.558-7.539A56.59,56.59,0,0,0,23.954,15.826Zm-7.068,17.65a.742.742,0,0,0,.7-.382.75.75,0,0,0-.041-.8l-5.81-8.267a15.393,15.393,0,0,1,4.8-2.687,13.063,13.063,0,0,0,1.784,2.4V27.83a.7.7,0,0,0,.013.137l.006.032a.735.735,0,0,0,.087.472l2.154,11.612-6.2-6.766Zm7.067-.8-4.11-4.817-.019-.1V25.039a6.857,6.857,0,0,0,8.218.029v2.686l-.031.151Zm3.106,7.378,2.353-11.517a.73.73,0,0,0,.108-.529l.006-.032a.682.682,0,0,0,.016-.15V23.783a13.008,13.008,0,0,0,1.8-2.393,15.307,15.307,0,0,1,4.672,2.639L30.2,32.3a.75.75,0,0,0-.041.8.739.739,0,0,0,.7.382l2.479-.157Z"/><path d="M14.641,10.984a33.568,33.568,0,0,0,9.313,1.224,31.645,31.645,0,0,0,9.164-1.227.749.749,0,0,0,.509-.807C32.376.49,28.88,0,28.191,0a8.453,8.453,0,0,0-4.316,1.141A8.453,8.453,0,0,0,19.559,0c-.689,0-4.184.49-5.436,10.174A.75.75,0,0,0,14.641,10.984ZM19.559,1.5a6.663,6.663,0,0,1,3.846,1.127.757.757,0,0,0,.936,0,6.666,6.666,0,0,1,3.85-1.13c1.822,0,3.3,4.615,3.857,8.229a32.713,32.713,0,0,1-8.094.979,34.417,34.417,0,0,1-8.25-.985C16.19,6.674,17.589,1.5,19.559,1.5Z"/><text x="0" y="63" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">Created by Koson Rattanaphan</text><text x="0" y="68" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">from the Noun Project</text></svg>
```

Comment:
Have you tested this on stage?

Reply:
No I will do it today

Reply:
Just a minor thing and it didn't work before. I tested on my local